homeassistant/components/august/__init__.py

```diff
@@ -75,138 +57,154 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-AUGUST_COMPONENTS = ["camera", "binary_sensor", "lock"]
 
+async def async_request_validation(hass, config_entry, august_gateway):
```

Comment:
For a future PR: We should use a config flow for this. A config flow can result in an update to an existing entry .

Reply:
added a comment